env/bulid.go

```diff
@@ -0,0 +1,46 @@
+package env
+
+var BuildEnvWhitelist = []string{
+	"CNB_STACK_ID",
+	"HOSTNAME",
+	"HOME",
+}
+
+func NewBuildEnv(environ []string) *Env {
+	return &Env{
+		RootDirMap: POSIXBuildEnv,
```

Comment:
Do these fields need to be exported?

Reply:
If we don't put the tests in the `env` package allow us to test all of the behavior. I know there are some work arounds but maybe we can come back to that later.